src/converse-rosterview.js

```diff
@@ -22,6 +23,26 @@ import tpl_search_contact from "templates/search_contact.html";
 const { Backbone, Strophe, $iq, b64_sha1, sizzle, _ } = converse.env;
 const u = converse.env.utils;
 
+// this is copied from converse-chatbowviews.js
+const AvatarMixin = {
+
+    renderAvatar (el) {
+        el = el || this.el;
+        const canvas_el = el.querySelector('canvas');
+        if (_.isNull(canvas_el)) {
+            return;
+        }
+        const image_type = this.model.vcard.get('image_type'),
+                image = this.model.vcard.get('image');
+
+        canvas_el.outerHTML = tpl_avatar({
+            'classes': canvas_el.getAttribute('class'),
+            'width': canvas_el.width,
+            'height': canvas_el.height,
+            'image': "data:" + image_type + ";base64," + image,
+        });
+    },
+};
```

Comment:
There's no need to duplicate the code. Instead, in `src/converse-chatboxviews.js`, you can set `AvatarMixin` as an attribute on the `_converse` object.

Then further down you can do:

```_converse.ViewWithAvatar = Backbone.NativeView.extend(_converse.AvatarMixin);```